oio/event/consumer.py

```diff
@@ -223,16 +223,22 @@ def handle(self, beanstalk):
                     eventlet.sleep(BEANSTALK_RECONNECTION)
                     continue
                 event = self.safe_decode_job(job_id, data)
-                try:
-                    self.process_event(job_id, event, beanstalk)
-                except (ClientException, OioNetworkException) as exc:
+                if not event:
                     self.logger.warn("Burying event %s (%s): %s",
-                                     job_id, event.get('event'), exc)
-                    beanstalk.bury(job_id)
-                except Exception:
-                    self.logger.exception("Burying event %s: %s",
-                                          job_id, event)
+                                     job_id, event.get('event'),
+                                     "malformed")
```

Comment:
Seems like a _Bad Idea ©_ to call `event.get()` after `if not event`.

Reply:
Thank you!